va.data-commons.org/portal/gitops.json

```diff
@@ -1,6 +1,6 @@
 {
   "gaTrackingId": "G-MNK00L5EG7",
-  "argoTemplate": "gwas-template-new-bucket",
```

Comment:
where can I find this template? I'm looking here, but it is not obvious to me where to look: https://github.com/uc-cdis/vadc-genesis-cwl/blame/master/argo/gwas-workflows/argo-genesis-association-uniflow.template.yml

Reply:
Thanks @vzpgb . Related question: what is https://github.com/uc-cdis/vadc-genesis-cwl/blob/master/argo/gwas-workflows/argo-genesis-association.template.yml ? Is this a previous version that we don't use anymore?